rollup.config.mjs

```diff
@@ -7,17 +7,15 @@ export default {
   input: "./src/index.ts",
   output: {
     file: './dist/types.d.ts',
-    format: 'es', // module format doesn't really matter here since output i
+    format: 'es', // module format doesn't really matter here since output is types
+  },
+  // function indicating which deps should be considered external: external deps will NOT have their types bundled
+  external: (id) => {
+    // bundle everything but '@openfeature/core', which is a peer
+    return id === '@openfeature/core';
   },
-  external: [
-    // function indicating which deps should be considered external: non-external deps will have their types bundled
-    (id) => {
-      // bundle 'events' types
-      return id !== 'events';
-    }
-  ],
   plugins: [
     // use the rollup override tsconfig (applies equivalent in each sub-packages as well)
-    dts({tsconfig: './tsconfig.rollup.json'}),
+    dts({tsconfig: './tsconfig.rollup.json', respectExternal: true }),
```

Comment:
The docs mention that `respectExternal` is not recommended. Can you explain why this is required?

https://github.com/Swatinem/rollup-plugin-dts?tab=readme-ov-file#what-to-expect

Reply:
It's what allows the `external` classifier above to work at all now.

If we don't `respectExternal`, it doesn't matter what we put in the function above - everything is considered external unless it's a relative import. This would result in the vents being bundled, as well as `core` being bundled, which we do not want.

Reply:
Makes sense!